examples/viewer.js

```diff
@@ -257,26 +261,28 @@ async function startViewer(localView, remoteView, formValues, onStatsReport, rem
         viewer.localView = localView;
         viewer.remoteView = remoteView;
 
-        if (formValues.enableProfileTimeline) {
-            viewer.remoteView.addEventListener('loadeddata', () => {
-                metrics.viewer.ttff.endTime = Date.now();
-
+        viewer.remoteView.addEventListener('loadeddata', () => {
```

Comment:
Would it work if we only listened to this event if either DQP or profile timeline is enabled?

Reply:
We are listening to this event regardless and handling DQP/profling accordingly. Did it this way to avoid duplication of event listening.